include/WorkingSetSelectorComponent.hpp

```diff
@@ -32,6 +32,11 @@ class WorkingSetSelectorComponent : public Component
 	void redraw();
 	void updateIopLoadIndicators();
 
+	static constexpr int WIDTH = 96;
+	static constexpr int BUTTON_WIDTH = 72;
+	static constexpr int BUTTON_HEIGHT = 72;
+	static int buttonPadding();
```

Comment:
Since `WIDTH` and `BUTTON_WIDTH` and everything else in `buttonPadding` is known at compile time, you can probably make the whole function `constexpr` so its computed at compile time

Reply:
Also, Nit (naming consistency)
```suggestion
	static int button_padding();
```